platform/test/server.test.ts

```diff
@@ -83,4 +83,46 @@ describe("Python Services", () => {
       });
     });
   });
+
+  describe("Error handling", () => {
```

Comment:
TODO for me: 
1. Test this locally
2. Think about whether `test_errors` is the best approach